posts/inside-rust/2024-08-22-embedded-wg-micro-survey.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Embedded Working Group Community Micro Survey"
+author: James Munns
+team: Embedded Devices Working Group <https://www.rust-lang.org/governance/wgs/embedded>
+---
+
+The [Embedded devices working group] has launched the [2024 Community Micro Survey] starting
```

Comment:
Let's make this a bit more explicit, `Community Micro Survey` on its own sounds a bit like it's some smaller version of the main Rust survey (which we also sometimes call "community survey"). Could you include the word "embedded" in the survey name mentioned here?

Reply:
Will do!

Reply:
@Kobzol hopefully addressed in https://github.com/rust-lang/blog.rust-lang.org/pull/1384/commits/8cc517d3f1427a7e9f9c42e624ed1a9be9c035f6 !